### DISCUSSION TITLE
Force refresh of conditionnal UI

### DISCUSSION THREAD

**Discussion by NicolasFabre81** [Budibase/budibase#12547] [Help] [answered]
I'm looking for a tip to force the refresh of conditionnal UI.

My rules are related to a data provider that is inside a repeater, so I can't refresh the data provider...

**Comment by melohagan** [comment]
Hey @NicolasFabre81 

Just to clarify, are you looking to force a refresh of a data provider? Or are you trying to force a refresh of a condition? 

If you have a filter on a data provider that uses a binding then you could potentially "trick" it into refreshing by updating the value.

For example you could have a filter:

> Filter: AutoID larger than State.RefreshCounter

Then you would just make sure to *decrement* the refresh counter so that the filter will always be true, but trigger a refresh when the counter value changes.

**Reply by NicolasFabre81** [reply, marked as answer]
I was trying to force the refresh of the condition itself, not the DP as I couldn't reach it.
But finally I found the solution : I binded this DP not on my table but on another DP at the top of the page, which I can reach and refresh !

It works perfectly, thanks for your support !

**Reply by melohagan** [reply]
Ah yes nice one! It is true that refreshing a parent data provider will cause cascading refreshing among the child data providers
